### PR TITLE
Refactor for glob support

### LINKED ISSUE #93: Glob input?
Hey, love this. I thought, "I wonder if there's a tool to check for unused exports", and there it was!

I was wondering if you'd accept a PR to change the path input to accept glob patterns instead of direct paths? I've tried to use my shell's glob expansion but it's not expressive enough for what I need.

Thanks!

### PR DESCRIPTION
**Pull request by benadamstyles**
I made a couple of fairly big changes! Firstly, I had to make `exec()` async because the glob package I used is async. Secondly, I changed how the jest snapshots were rendered, just so it was easy to check visually that they output was what I expected. Let me know if you want any of this changed!

Also there's both a `yarn.lock` and a `package-lock.json`, so I ran `yarn` and `npm i` to update both!

Fixes #93

### CODE REVIEW

**Comment by benadamstyles** [comment]
Fixes #93

**Comment by benadamstyles** [comment]
Ah ok, tests fail because `async` is not supported until node `v7.6.0`. Should I use promises, or raise the minimum node version?

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "@babel/traverse": "7.1.4",
     "chalk": "2.4.1",
     "cheerio": "1.0.0-rc.2",
+    "globby": "^8.0.1",
```
[the remark this answers is not in the document]
Comment (benadamstyles) [reply package.json]: Done!

**Comment by benadamstyles** [comment]
Ah right, I've hit this problem before. For some reason, eslint skips from node v6 to v8 in its compatibility list, so v7 isn't allowed. Should I therefore raise the minimum node version further, to v8?

**Comment by yamafaktory** [comment]
@benadamstyles Alright, yeah I faced that issue too in some projects. Let's go for it 🚀.

**Comment by benadamstyles** [comment]
Haha ok now the object is being rendered to a snapshot with keys in a different order in node 8 vs node latest! I'll revert to just testing on node 8.

**Comment by benadamstyles** [comment]
Ok all passing! Had to sort the paths returned by `globby` to stabilise the order of shrimpit results.

**Comment by yamafaktory** [comment]
Awesome work @benadamstyles 👍. Closes #93.

**Pull request merged**